src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -351,6 +351,13 @@ export default {
 			if (this.$route.query.page && Number(this.$route.query.page) > 1) {
 				url = `${url}?page=${this.$route.query.page}`;
 			}
+			// MARS-310 Category Share on Thanks page
```

Comment:
@dyersituations this function overrides the canonical url for `lend-by-category` page, didn't notice after a couple of tests